tasks.py

```diff
@@ -436,7 +436,7 @@ def nbshell(context):
     """Launch an interactive Nautobot shell."""
     command = "nautobot-server nbshell"
 
-    run_command(context, command, pty=True)
+    run_command(context, command)
```

Comment:
I'd fixed this a different way locally, by changing `run_command`:

```diff
-        return docker_compose(context, compose_command, pty=True, **kwargs)
+        kwargs["pty"] = True
+        return docker_compose(context, compose_command, **kwargs)
```

Thoughts on one versus the other?

Reply:
I don't think it matters either way